WalletLibrary/WalletLibrary/Requests/Handlers/OpenIdRequestProcessor.swift

```diff
@@ -11,10 +11,9 @@ enum OpenIdRequestHandlerError: Error
 }
 
 /**
- * Handles a raw Open Id request and configures a VeriifedIdRequest object.
- * Post Private Preview TODO: add processors to support multiple profiles of open id.
+ * Processes a raw Open Id request and configures a VeriifedIdRequest object.
  */
-public struct OpenIdRequestProcessor: RequestProcessing
+struct OpenIdRequestProcessor: RequestProcessing
```

Comment:
This one made public so the FaceCheckExtension could list it as its supported RequestProcessor.

Reply:
Talked offline, going to play around in the next PR to see if we can make most of this internal.